framework/console/controllers/MessageController.php

```diff
@@ -386,7 +386,7 @@ protected function saveMessagesToDb($messages, $db, $sourceMessageTable, $messag
 
         if (!$removeUnused) {
             foreach ($obsolete as $pk => $msg) {
-                if (mb_substr($msg, 0, 2) === '@@' && mb_substr($msg, -2) === '@@') {
+                if (strpos($msg, '@@') === 0 && substr($msg, -2) === '@@') {
```

Comment:
I think these shoud be kept multibyte-aware.

Reply:
See my comment from above. And I have tested `substr($msg, -2) === '@@'` with some arabic text.